combinations/hash.tsv

```diff
@@ -98,3 +98,7 @@ qcat=1.1.0,pigz=2.3.4	bgruening/busybox-bash:0.1	0
 r-base=4.0.2,r-devtools=2.3.2,r-gmodels=2.18.1,r-optparse=1.6.6,bioconductor-deseq2=1.30.0,bioconductor-edger=3.32.0,bioconductor-genomicalignments=1.26.0,bioconductor-org.hs.eg.db=3.12.0,bioconductor-rsubread=2.4.0
 r-base=4.0.3,r-reshape2=1.4.4,r-optparse=1.6.6,r-ggplot2=3.3.3,r-scales=1.1.1,r-viridis=0.5.1,r-tidyverse=1.3.0,bioconductor-biostrings=2.58.0,bioconductor-complexheatmap=2.6.2
 metabat2=2.15,python=3.6.7,biopython=1.74,pandas=1.1.5
+cd-hit=4.8.1,exonerate=2.4,blast=2.2.31
+trinity=2.9.1,pigz=2.3.4
+python=3.6.10,biopython=1.78,pandas=1.1.2,numpy=1.18.1
+r-base=3.6.2,r-reshape2=1.4.4,r-plotly=4.9.2.1,plotly-orca=3.4.2,r-ggplot2=3.3.0,r-svglite=1.2.3,r-ggthemes=4.2.0,r-knitr=1.29,r-rmarkdown=2.3,r-kableextra=1.1.0
```

Comment:
r-base does not need to be specified here

Reply:
Ok. Will remove the r-base. Thanks